go.mod

```
@@ -178,6 +179,9 @@ require (
 
 // TODO: Once the repo is public, remove this so we use a versioned module
 replace (
+	github.com/aws/eks-anywhere/internal/aws-sdk-go-v2/internal/configsources => ./internal/aws-sdk-go-v2/internal/configsources
+	github.com/aws/eks-anywhere/internal/aws-sdk-go-v2/internal/endpoints/v2 => ./internal/aws-sdk-go-v2/internal/endpoints/v2
+	github.com/aws/eks-anywhere/internal/aws-sdk-go-v2/service/snowballdevice => ./internal/aws-sdk-go-v2/service/snowballdevice
```

Can't we just do a replace for the whole module?
```suggestion
	github.com/aws/eks-anywhere/internal/aws-sdk-go-v2 => ./internal/aws-sdk-go-v2
```

---

there a 3 submodules under `internal/aws-sdk-go-v2` thats why i replace all